### PR TITLE
[DEV-2570] integrate services component into SELL and BUY tab

### PR DESCRIPTION
**Pull request by lapatric**
+ Swap tab implemented

### CODE REVIEW

File: src/components/tabs/sell.tab.tsx [hunk 1]
```diff
@@ -33,15 +37,78 @@ export function useSellTab(): StyledTabProps {
...
+  const { authenticationToken } = useAuthContext();
+  const { sync } = useSessionContext();
+
+  useEffect(() => {
+    if (authenticationToken) {
+      if (step !== SellTabStep.SELL_PROCESS) setStep(SellTabStep.SELL_PROCESS);
+    } else {
+      if (step !== SellTabStep.OVERVIEW) setStep(SellTabStep.OVERVIEW);
+    }
+  }, [authenticationToken]);
```
Comment (davidleomay) [inline src/components/tabs/sell.tab.tsx:56]: Does this work with WalletConnect that we directly login if there is an authentication token?

**Review by davidleomay — COMMENTED** [review]
- [x] Update Github secrets

File: src/components/tabs/sell.tab.tsx
Comment (davidleomay) [inline src/components/tabs/sell.tab.tsx:1]: Build is failing

File: src/components/tabs/sell.tab.tsx (hunk 1 above)
Comment (lapatric) [reply src/components/tabs/sell.tab.tsx:56]: Yes, based on my tests it works well for login. However, I've not tested completing a transaction to see whether the connection to the wallet can be reestablished, so we still have to do that.

**Review by davidleomay — APPROVED** [review]

**Pull request merged**